stripe/src/main/java/com/stripe/android/view/PaymentMethodsActivity.java

```diff
@@ -182,6 +185,20 @@ private void updatePaymentMethods(@NonNull List<PaymentMethod> paymentMethods,
         if (selectPaymentMethodId != null) {
             mMaskedCardAdapter.setSelectedPaymentMethod(selectPaymentMethodId);
         }
+        
+         Customer arr = mCustomerSession.getCachedCustomer();
+        int selectedIndex = 0;
+        String defaultSource = arr.getDefaultSource();
+        for (int j = 0; j < paymentMethods.size(); j++) {
+            if (defaultSource.equalsIgnoreCase(paymentMethods.get(j).id)) {
```

Comment:
This expression will never be true, because defaultSource will never have the value of a PaymentMethod id